cmd/flag/open_stream_flags.go

```diff
@@ -17,15 +17,32 @@ package flag
 
 import (
 	"github.com/spf13/cobra"
+	"time"
+)
+
+var (
+	defaultEnableAutoClose = false
+	defaultAutoCloseDelay  = 5 * time.Second
+	defaultAutoCloseTime   = "2006-01-02'T'15:04:05Z"
 )
 
 type OpenStreamFlag struct {
-	StreamId string
+	EnableAutoClose bool
+	AutoCloseDelay  time.Duration
+	AutoCloseTime   string
+	StreamId        string
 }
 
 // Add a flag to the command.
 func (f *OpenStreamFlag) AddFlags(cmd *cobra.Command) {
 	cmd.Flags().StringVarP(&f.StreamId, "stream-id", "r", "", "The StreamId to resume.")
+	cmd.Flags().BoolVarP(&f.EnableAutoClose, "enable-auto-close", "", defaultEnableAutoClose,
+		"When set to true, the stream will close after a specified auto close time.")
+	cmd.Flags().DurationVarP(&f.AutoCloseDelay, "auto-close-delay", "", defaultAutoCloseDelay,
```

Comment:
I think this is in line with what @chelliwell mentioned in another comment, but we should describe how durations are defined.

Some docs about durations in golang are here: https://golang.org/pkg/time/#ParseDuration, but it's not fair to assume users of stellarcli know about golang, so we should briefly describe likely durations (seconds and minutes).

Reply:
Added a brief summary of how durations are used here.